### PR TITLE
fix: change css class names for logo animation

### PR DESCRIPTION
**Pull request by e-aakash**
`p` is very common css class, and causing some problems (placement blog entries)
Current fix: change p<num> to logo<num>

### CODE REVIEW

**Comment by e-aakash** [comment]
This is not working on local testing :/

File: src/index.html [hunk 1]
```diff
@@ -78,21 +78,21 @@
...
+    .logo1 { transform: rotate(67deg); }
+    .logo2 { transform: rotate(45deg); }
+    .logo3 { transform: rotate(20deg); }
+    .logo4 { transform: rotate(00deg); }
+    .logo5 { transform: rotate(-20deg); }
+    .logo6 { transform: rotate(-45deg); }
+    .logo7 { transform: rotate(-67deg); }
+
+    .loading-fader .logo7 { -webkit-animation-delay: 0.09s; animation-delay: 0.09s; }
+    .loading-fader .lgoo6 { -webkit-animation-delay: 0.18s; animation-delay: 0.18s; }
```
Comment (pulsejet) [inline src/index.html]: Wtf

**Review by pulsejet — APPROVED** [review]
Can change class names to instiapp-logo-1 etc, LGTM otherwise

File: src/index.html (hunk 1 above)
Comment (e-aakash) [reply src/index.html]: that is called a typo

**Pull request merged**